.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
```

Comment:
Why do we need this file now?

Reply:
Was adding CI via circle, however, I'm removing it